src/App.tsx

```diff
@@ -1,204 +1,72 @@
-              />
-            </form>
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorNotification(Errors.LOADING);
+      });
+  }, []);
 
-            <div data-cy="TodoLoader" className="modal overlay">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+  useEffect(() => {
+    setTimeout(() => {
+      setErrorNotification(Errors.DEFAULT);
+    }, 3000);
+  });
 
-          {/* This todo is in loadind state */}
-          <div data-cy="Todo" className="todo">
-            <label className="todo__status-label">
-              <input
-                data-cy="TodoStatus"
-                type="checkbox"
-                className="todo__status"
-              />
-            </label>
+  const activeTodosCount = todos.filter(todo => !todo.completed).length;
 
-            <span data-cy="TodoTitle" className="todo__title">
-              Todo is being saved now
-            </span>
+  const getFilteredTodos = () => {
+    switch (filter) {
+      case Filter.ACTIVE:
+        return todos.filter(todo => !todo.completed);
 
-            <button type="button" className="todo__remove" data-cy="TodoDelete">
-              ×
-            </button>
+      case Filter.COMPLETED:
+        return todos.filter(todo => todo.completed);
 
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
-        </section>
+      default:
+        return todos;
+    }
+  };
 
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
+  const filteredTodos = getFilteredTodos();
 
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
+  return (
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
 
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
+      <div className="todoapp__content">
+        <Header />
 
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
-            >
-              Completed
-            </a>
-          </nav>
+        {todos.length && <TodoList todos={filteredTodos} />}
```

Comment:
conditional rendering is good, but make sure your condition is boolean, because now it looks like this if there are no todos
![image](https://github.com/user-attachments/assets/0cd585d0-1ae7-49ad-841c-483af1e402f5)